### PR TITLE
Move event logging to debug

### PR DESCRIPTION
**Pull request by shosti**
@rainforestapp/core @ukd1

### CODE REVIEW

**Comment by ukd1** [comment]
@shosti this is mega failing CI - plz fix

File: lib/simple_sqs/processor.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def process_sqs_message sqs_message
 
   private
   def process event
-    logger.info "Processing SQS event #{event.inspect}"
+    logger.debug "Processing SQS event #{event.inspect}"
```
Comment (jipiboily) [inline lib/simple_sqs/processor.rb:25]: Oops, my bad! This is my fault! It must fill the logs like crazy 😉

**Comment by shosti** [comment]
@ukd1 OK build fixed. Look OK?

**Review by nxvl — APPROVED** [review]
🚢

**Pull request merged**